### ISSUE TITLE
INS node position resolution missing

### ISSUE THREAD

**Issue by platisd** [platisd/indoor-navigation-system#87]
## Description
As an INS node I should instruct the server to resolve my position after I am done transmitting the RSSI values. On how to do that, please refer to `Trigger position calculation for device` section of the [server/README.md](https://github.com/platisd/indoor-navigation-system/blob/dev/server/README.md).
also: ask the server to resolve position some fixed period

## Definition of Done
* The `ins-node` [WiFi module](https://github.com/platisd/indoor-navigation-system/blob/dev/ins-node/src/wifi_module/wifi_module.ino) is able to trigger the position resolution interface of the server after successfully transmitting a batch of data.
* The added functionality is tested.

**Comment by lexious89** [comment]
+ and in addition ask the server to resolve position after some fixed period (say every minute) even if the device node does not request it.

**Comment by platisd** [comment]
Do you mean to have a timer on the server? I think it's supposed to be stateless. What do you think @samup4web

**Comment by lexious89** [comment]
Yes, it is stateless in relation to the ins nodes. But the server knows how long it has been running. 
This is needed because if it is the first time the insnode is transmitting data and it happens to loose connection or something goes wrong so that the location table in ins_db is empty for that device, any front end get-position request will yield unpredictable results - a nan in the localization process, whilst in fact the data is available for localization.

**Comment by platisd** [comment]
Sounds good to me
